halo2_proofs/src/dev.rs

```diff
@@ -1040,7 +1099,16 @@ impl<F: FieldExt> MockProver<F> {
                                 row,
                             })
                         }
-                    })
+                    };
+                    let indexes: Vec<usize> = (0..values.len()).into_iter().collect();
+                    let ret: Vec<VerifyFailure> = indexes
+                        .par_iter()
```

Comment:
Should this `par_iter()` depend on `ISPAR`? Maybe it needs its own `home_made_par_iter_filter_map` wrapper.